deployment/src/main/java/com/github/mcollovati/quarkus/hilla/deployment/asm/SpringReplacementsClassVisitor.java

```diff
@@ -15,14 +15,43 @@
  */
 package com.github.mcollovati.quarkus.hilla.deployment.asm;
 
+import java.util.Map;
+
+import dev.hilla.AuthenticationUtil;
 import io.quarkus.gizmo.Gizmo;
 import org.objectweb.asm.ClassVisitor;
 import org.objectweb.asm.MethodVisitor;
 
+import com.github.mcollovati.quarkus.hilla.SpringReplacements;
+
+/**
+ * This ClassVisitor searches for used methods which are based on Spring and replaces them with a Quarkus equivalent.
+ */
 public class SpringReplacementsClassVisitor extends ClassVisitor {
 
+    private static final Map<MethodSignature, MethodSignature> STATIC_SPRING_REDIRECTS = Map.of(
```

Comment:
nit: `STATIC_` seems a bit redundant
```suggestion
    private static final Map<MethodSignature, MethodSignature> SPRING_REDIRECTS = Map.of(
```

Reply:
The static is not meant for static field, it's meant for the static spring methods we replace. So that it is clear that the replaced methods are statics. But if it is not clear we can drop the prefix

Reply:
Got it. What about `SPRING_STATIC_METHOD_REDIRECTS`?

Reply:
Sounds good 👍🏻